### DISCUSSION TITLE
Will Robolectric continus to maintain Chrome extension?

### DISCUSSION THREAD

**Discussion by utzcoz** [robolectric/robolectric#7102] [Q&A] [answered]
I have saw some folks want Robolectric Chrome extension to check Shadow APIs when reviewing Android SDK API reference. But Chrome extension is not accessed now, what about making it alive again? /cc @hoisie .

**Comment by hoisie** [comment, marked as answer]
To be honest I have no idea what the state of the chrome extension is. I do not have the access or credentials to make updates. I personally have not had much desire to use it, and to me it may be a maintenance burden because you will have to update it if the html layout of the Android docs changes. So I'd personally be in favor of dropping it completely.

**Reply by utzcoz** [reply]
What about removing related information and links from robolectric.org?

**Reply by utzcoz** [reply]
The Chrome extension info was removed from robolectric.org contributing guide page. We should check whether there are other pages have related information.
